tests/functional/package_test/__init__.py

```diff
@@ -256,22 +251,11 @@ def _generate_skip_list_file(skip_list_file):
 def _run_check(shared_test_params, skip_list_file, test_project_build_cmd,
                test_project_name, test_project_path):
     """Check a test project."""
-    check_cmd = []
-    check_cmd.append('CodeChecker')
-    check_cmd.append('check')
-    check_cmd.append('-w')
-    check_cmd.append(shared_test_params['workspace'])
-    check_cmd.append('--suppress')
-    check_cmd.append(shared_test_params['suppress_file'])
-    check_cmd.append('--skip')
-    check_cmd.append(skip_list_file)
-    check_cmd.append('-n')
-    check_cmd.append(test_project_name)
-    check_cmd.append('-b')
-    check_cmd.append("'"+test_project_build_cmd+"'")
-    check_cmd.append('--analyzers')
-    check_cmd.append('clangsa')
-    check_cmd.append('--quiet-build')
+    check_cmd = ['CodeChecker', 'check', '-w', shared_test_params['workspace'],
```

Comment:
I'm not sure this is more readable than before. I would much prefer if a list is used to move each argument into a new line.
```
check_cmd = ['CodeChecker',
             'check',
             '-w', shared_test_params['workspace'],
             ....
```